manuscript/manuscript.Rmd

```diff
@@ -96,19 +96,19 @@ the number of figures, tables, and text boxes: **6** (five figures, 1 table)
 <!-- Environmental problems, ML -->
 Advances in both available data and computing power are opening the door for machine learning (ML) to play a greater role in addressing some of our planet’s most pressing environmental problems. 
 But will ML approaches really help us tackle our most pressing environmental problems? From the growing frequency and intensity of wildfire [@wildfire], to over-exploited fisheries [@Worm2006] and declining biodiversity [@biodiversity], to emergent zoonotic pandemics [@covid], the diversity and scope of environmental problems are unprecedented.
-Applications of ML in ecology have to-date illustrated the promise of two methods: *supervised learning* [@joseph_neural_2020] and *unsupervised learning* [@valletta_applications_2017]. However, the fields of ecology and conservation science have so far overlooked the third and possibly most promising approach in the ML triad: *reinforcement learning* (RL).
+Applications of ML in ecology have to-date illustrated the promise of two methods: *supervised learning* [@joseph_neural_2020] and *unsupervised learning* [@valletta_applications_2017]. However, the fields of ecology and conservation science have largely overlooked the third and possibly most promising approach in the ML triad: *reinforcement learning* (RL).
 Three features distinguish RL from other ML methods in ways that are particularly well suited to addressing issues of global ecological change:
 
 1) RL is explicitly focused on the task of selecting actions in an uncertain and changing environment to maximize some objective,
 2) RL does not require massive amounts of representative sampled historical data,
 3) RL approaches easily integrate with existing ecological models and simulations, which may be our best guide to understanding and predicting future possibilities.
 
-Despite relevance to decision making under uncertainty that could make RL uniquely well suited for ecological and conservation problems, it has so far seen little application in these fields.
+Despite relevance to decision making under uncertainty that could make RL uniquely well suited for ecological and conservation problems, RL has not been widely applied across these fields except in behavioral ecology [@yamada_evolution_2020, @wang_reinforcement_2020].
```

Comment:
I think we should replace "ecological and conservation" with "ecological conservation".  That makes it more explicit that the behavioral ecology is not part of our scope.

I think we could have a second sentence that acknowledges the behavioral ecology citations while also observing that this involves very different assumptions; maybe like this:

> While Deep RL has started to see some applications in behavioral ecology [@, @], it is important to note that this requires much stronger assumptions than applications in the conservation context we focus on here. In conservation problems, Deep RL serves only the role of an optimization algorithm which may be able to discover suitably clever policies that human managers would not be able to find without it.  This is very different than assuming that the learning process utilized by any given Deep RL algorithm is an accurate model of underlying biological process of learning or evolution in any given species.

Or maybe that ought to more delicately worded?

Reply:
Sketching this out I am concerned that writing fully on behav. eco. here might take away from general theme of this paragraph. I'll try to put something together in next commit or so.

Reply:
Good call, most of this should wait for the discussion section.  

I think if the intro is clear enough about our focus on conservation / management / global change decision problems, we could even get away with not bringing up ecology explicitly until the discussion?